### PR TITLE
open-cluster-management/backlog#8328--credentials

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Comment by dockerymick** [comment]
Verified that the links are working. I will review again when more info is added

File: security/credentials.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+[#cloud-provider-credentials]
+== Credentials
+
+
+[#aws-rotate]
+=== Amazon Web Services
+
+. View the resources in the namespace that has the same name as the cluster with the expired credential. 				
+. Find the secret name `<cluster_name>-<cloud_provider>-creds`. For example, `my_cluster-aws-creds1.
```
Comment (dockerymick) [inline security/credentials.adoc]: missing tick mark to close monospace

File: security/credentials.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+
+[#aws-rotate]
+=== Amazon Web Services
+
+. View the resources in the namespace that has the same name as the cluster with the expired credential. 				
+. Find the secret name `<cluster_name>-<cloud_provider>-creds`. For example, `my_cluster-aws-creds1.				
+. Edit the secret to replace the existing value with the updated value.				
+. View the resources in the namespace that has the same name as the cluster with the expired credential. 				
+. Find the secret name `<cluster_name>-<cloud_provider>-creds`. For example, `my_cluster-aws-creds1.
```
Comment (dockerymick) [inline security/credentials.adoc]: same here, missing ending tick mark

File: security/credentials.adoc (hunk 1 above)
Comment (swopebe) [reply security/credentials.adoc]: Wasn't done lol, I think I caught it.

File: security/credentials.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+See the procedure to rotate the following credentials for your cluster:
+
+  - aws_access_key_id
+  - aws_secret_access_key
+
+. View the resources in the namespace that has the same name as the cluster with the expired credential. 				
+. Find the secret name `<cluster_name>-<cloud_provider>-creds`. For example: `my_cluster-aws-creds1`.				
+. Edit the secret to replace the existing value with the updated value.	
+		
+//[#gke-platform]
```
Comment (swopebe) [inline security/credentials.adoc:33]: these are commented out because I don't have the info.

**Review by StacyCannon — APPROVED** [review]
LGTM.  I'd recommend to have Jon do a quick review on Thursday just to make sure there wasn't something specific he was looking for.

File: security/credentials.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+[#rotating-credentials]
+== Provider credentials
+
+[#aws-rotate]
+=== Amazon Web Services
+
+See the procedure to rotate the following credentials for your cluster:
+
+  - aws_access_key_id
```
Comment (dockerymick) [inline security/credentials.adoc]: Should the credentials be monospaced?

File: security/credentials.adoc (hunk 3 above)
Comment (jnpacker) [reply security/credentials.adoc:33]: We don't do gke with credentials.

File: security/credentials.adoc [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  - aws_secret_access_key
+
+. View the resources in the namespace that has the same name as the cluster with the expired credential. 				
+. Find the secret name `<cluster_name>-<cloud_provider>-creds`. For example: `my_cluster-aws-creds1`.				
+. Edit the secret to replace the existing value with the updated value.	
+		
+//[#gke-platform]
+//=== Google Cloud Platform
+
```
Comment (jnpacker) [inline security/credentials.adoc:35]: GCP credential key names"
- gcServiceAccountKey  (There is info when creating the credential how to get the new value.

File: security/credentials.adoc [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+//[#microsoft-azure]
+//=== Microsoft Azure 
+
```
Comment (jnpacker) [inline security/credentials.adoc:58]: - clientId
- clientSecret

I don't expect tenantId or subscriptionId to change. If you have new versions of these, you'll need a new Cloud provider created (point back to the creating a cloud provider doc)

File: security/credentials.adoc [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+
```
Comment (jnpacker) [inline security/credentials.adoc:62]: //=== VMware vSphere
- username
- password
- cacertificate

File: security/credentials.adoc (hunk 6 above)
Comment (swopebe) [reply security/credentials.adoc:58]: @jnpacker I added these under the azure section, but how does the user rotate them?

File: security/credentials.adoc (hunk 7 above)
Comment (swopebe) [reply security/credentials.adoc:62]: How does the user rotate these?

File: security/credentials.adoc (hunk 3 above)
Comment (swopebe) [reply security/credentials.adoc:33]: ok removed

File: security/credentials.adoc (hunk 4 above)
Comment (swopebe) [reply security/credentials.adoc]: yes

File: security/credentials.adoc (hunk 7 above)
Comment (swopebe) [reply security/credentials.adoc:62]: commented out until we have more

File: security/credentials.adoc (hunk 6 above)
Comment (swopebe) [reply security/credentials.adoc:58]: commented out until we have more

File: security/credentials.adoc (hunk 5 above)
Comment (swopebe) [reply security/credentials.adoc:35]: commented out until we have more

**Review by jpweiser — APPROVED** [review]
LGTM

File: security/credentials.adoc [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+[#rotating-credentials]
+== Provider credentials
+
+Connection secrets for cloud provider can be rotated. See the following list of provider credentials:
+
+[#aws-rotate]
+=== Amazon Web Services
+
+  - `aws_access_key_id`
```
Comment (dockerymick) [inline security/credentials.adoc]: Should there be a description for these services, similar to the Agent section?

File: security/credentials.adoc [hunk 9]
```diff
@@ -0,0 +1,62 @@
+[#credentials]
+= Credentials
+
+You can rotate your credentials for your {product-title} clusters.
+
+[#rotating-credentials]
+== Provider credentials
+
+Connection secrets for cloud provider can be rotated. See the following list of provider credentials:
```
Comment (dockerymick) [inline security/credentials.adoc]: Connection secrets for a cloud provider

**Review by dockerymick — COMMENTED** [review]
I made a change and left a question for you

File: security/credentials.adoc (hunk 8 above)
Comment (swopebe) [reply security/credentials.adoc]: I will have to research this -- but can be added if I get it! Thanks

Comment (swopebe) [reply security/credentials.adoc]: done

**Pull request merged**